### ISSUE TITLE
nl2br is missing from rendering of 'multi-line text' fields in email notifications

### ISSUE THREAD

**Issue by thijskaspers** [verbb/formie#379]
**What are you trying to do?**
When using the default email notification template, the 'nl2br' filter is missing for (multiline) text fields. It is working when we render the field as rich text, but not when it's a default field without rich text formatting.

We believe it should always go through a 'nl2br' filter by default, because without it e-mails become a single line and that is of course harder to read for clients.

Note we are using the "All Form Fields" option to automatically render all fields in the email notification template.

**Additional context**
Formie version 1.3.20
Craft 3.6.8

**Comment by engram-design** [comment]
Just came across this yesterday, so appreciate the timely reminder. Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.21",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Should be fixed in [1.3.22](https://github.com/verbb/formie/releases/tag/1.3.22)
